scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -82,7 +82,7 @@ abstract class ScanResultsStorage {
         }
 
         private fun configureDefaultStorage() {
-            val localFileStorage = XZCompressedLocalFileStorage(getOrtDataDirectory().resolve("$TOOL_NAME/results"))
+            val localFileStorage = XZCompressedLocalFileStorage(getOrtDataDirectory().resolve("results"))
```

Comment:
This is a potentially breaking change for users that already have scan results stored in this directory and should be mentioned in the commit message.